buildSrc/src/main/groovy/io/micronaut/guides/GuideAsciidocGenerator.groovy

```diff
@@ -252,4 +288,32 @@ class GuideAsciidocGenerator {
                 return ''
         }
     }
+
+    private static String resolveTestExtension(TestFramework testFramework) {
+        switch (testFramework) {
```

Comment:
I think we have methods for this in starter: `Language:getExtension`

Reply:
Done